### PR TITLE
Add recipes for R 3.2.0 and 3.2.1

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by bgruening** [comment]
I think I will close this one here? https://github.com/galaxyproject/tools-iuc/pull/222

File: packages/package_r_3_2_0/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+categories:
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads and compiles version
+  3.2.0 of the R package.
+long_description: |
+  This package is configured with the following options:
+
+  ./configure --with-tcltk \
```
Comment (erasche) [inline packages/package_r_3_2_0/.shed.yml]: can we get rid of the extraneous newlines here @davebx?

**Comment by erasche** [comment]
Sounds good @bgruening

**Comment by blankenberg** [comment]
See also: https://github.com/davebx/tools-iuc/pull/1

**Comment by davebx** [comment]
Thanks, @blankenberg.

**Comment by nsoranzo** [comment]
What's the point of adding a recipe for R 3.2.0 if we have one for 3.2.1?

**Comment by davebx** [comment]
@nsoranzo This pull request was originally intended for 3.2.0, I chose not to remove that version when I created one for 3.2.1.

**Pull request merged**